+ source /root/mugen-riscv/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environment preparation.'
+ message='Start environment preparation.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start environment preparation.'
Sun Dec  4 08:55:33 2022 - INFO  - Start environment preparation.
+ DNF_INSTALL rpmdevtools
+ pkgs=rpmdevtools
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py install --pkgs rpmdevtools --node 1 --tempfile ''
+ tmpfile2=/tmp/tmp7i32eofi
+ '[' -z '' ']'
+ tmpfile=/tmp/tmp7i32eofi
++ dnf list
++ head -n 3
++ tail -n 1
++ awk '{print $1}'
++ awk 'BEGIN {FS="."} {print $1}'
+ pkg_name=NetworkManager
++ dnf list
++ head -n 3
++ tail -n 1
++ awk '{print $1}'
++ awk 'BEGIN {FS="."} {print $2}'
+ pkg_arch=riscv64
+ yumdownloader NetworkManager
oe_test_rpmdevtools_rpmargs.sh: line 26: yumdownloader: command not found
+ mkdir -p /ALT/Sisyphus/files/i586/RPMS
+ mkdir -p /ALT/Sisyphus/files/noarch/RPMS
+ mkdir -p /ALT/Sisyphus/files/SRPMS
+ cp 'NetworkManager*.rpm' /ALT/Sisyphus/files/SRPMS/
cp: cannot stat 'NetworkManager*.rpm': No such file or directory
+ cp 'NetworkManager*.rpm' /ALT/Sisyphus/files/i586/RPMS/
cp: cannot stat 'NetworkManager*.rpm': No such file or directory
+ cp 'NetworkManager*.rpm' /ALT/Sisyphus/files/noarch/RPMS
cp: cannot stat 'NetworkManager*.rpm': No such file or directory
++ dnf list
++ head -n 4
++ tail -n 1
++ awk '{print $1}'
++ awk 'BEGIN {FS="."} {print $1}'
+ pkg_name1=NetworkManager-libnm
+ mkdir ./tmp_dir
+ yumdownloader NetworkManager-libnm
oe_test_rpmdevtools_rpmargs.sh: line 36: yumdownloader: command not found
+ LOG_INFO 'End of environmental preparation.'
+ message='End of environmental preparation.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End of environmental preparation.'
Sun Dec  4 08:57:41 2022 - INFO  - End of environmental preparation.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Sun Dec  4 08:57:42 2022 - INFO  - Start to run test.
+ rpmargs -h
+ grep 'Usage:
  rpmargs'
Usage:
    rpmargs [-h] -c command [-a] [-p] [FILE...] [DIR...]
+ CHECK_RESULT 0 0 0 'Failed option: -h'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Failed option: -h'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ rpmargs -c file -a
+ grep NetworkManager
+ CHECK_RESULT 1 0 0 'Failed option: -a'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: -a'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: -a'
+ LOG_ERROR 'Failed option: -a'
+ message='Failed option: -a'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Failed option: -a'
Sun Dec  4 08:57:45 2022 - ERROR - Failed option: -a
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_rpmdevtools_rpmargs.sh line 47'
+ message='oe_test_rpmdevtools_rpmargs.sh line 47'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_rpmdevtools_rpmargs.sh line 47'
Sun Dec  4 08:57:46 2022 - ERROR - oe_test_rpmdevtools_rpmargs.sh line 47
+ return 0
+ rpmargs -c file -p '/ALT/Sisyphus/files/noarch/RPMS/NetworkManager*.rpm'
+ grep NetworkManager
readlink: '/ALT/Sisyphus/files/noarch/RPMS/NetworkManager*.rpm': No such file or directory
+ CHECK_RESULT 1 0 0 'Failed option: -p'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: -p'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: -p'
+ LOG_ERROR 'Failed option: -p'
+ message='Failed option: -p'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Failed option: -p'
Sun Dec  4 08:57:47 2022 - ERROR - Failed option: -p
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_rpmdevtools_rpmargs.sh line 49'
+ message='oe_test_rpmdevtools_rpmargs.sh line 49'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_rpmdevtools_rpmargs.sh line 49'
Sun Dec  4 08:57:48 2022 - ERROR - oe_test_rpmdevtools_rpmargs.sh line 49
+ return 0
+ rpmdev-checksig 'NetworkManager*.rpm'
+ grep 'NetworkManager.*.rpm: RSA/SHA1'
+ CHECK_RESULT 1 0 0 'Failed command:rpmdev-checksig'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed command:rpmdev-checksig'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed command:rpmdev-checksig'
+ LOG_ERROR 'Failed command:rpmdev-checksig'
+ message='Failed command:rpmdev-checksig'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Failed command:rpmdev-checksig'
Sun Dec  4 08:57:49 2022 - ERROR - Failed command:rpmdev-checksig
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_rpmdevtools_rpmargs.sh line 52'
+ message='oe_test_rpmdevtools_rpmargs.sh line 52'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_rpmdevtools_rpmargs.sh line 52'
Sun Dec  4 08:57:50 2022 - ERROR - oe_test_rpmdevtools_rpmargs.sh line 52
+ return 0
+ head -n 1
+ rpmdev-cksum 'NetworkManager*rpm'
+ awk '{print $3}'
+ grep 'NetworkManager.*rpm'
cksum: 'NetworkManager*rpm': No such file or directory
Error: file does not exist: '/root/mugen-riscv/testcases/cli-test/rpmdevtools/oe_test_rpmdevtools_rpmargs/NetworkManager*rpm'
+ CHECK_RESULT 1 0 0 'Failed command: rpmdev-cksum'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed command: rpmdev-cksum'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed command: rpmdev-cksum'
+ LOG_ERROR 'Failed command: rpmdev-cksum'
+ message='Failed command: rpmdev-cksum'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Failed command: rpmdev-cksum'
Sun Dec  4 08:57:52 2022 - ERROR - Failed command: rpmdev-cksum
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_rpmdevtools_rpmargs.sh line 55'
+ message='oe_test_rpmdevtools_rpmargs.sh line 55'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_rpmdevtools_rpmargs.sh line 55'
Sun Dec  4 08:57:53 2022 - ERROR - oe_test_rpmdevtools_rpmargs.sh line 55
+ return 0
+ rpmdev-diff -v
+ grep 'rpmdev-diff version'
rpmdev-diff version 1.4
+ CHECK_RESULT 0 0 0 'Failed option: -v'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Failed option: -v'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ rpmdev-diff -h
+ grep -A 5 rpmdev-diff
+ grep Options:
Options:
+ CHECK_RESULT 0 0 0 'Failed option: -h'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Failed option: -h'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ rpmdev-diff -c 'NetworkManager*.rpm' 'NetworkManager-libnm*.rpm'
Error: file does not exist: 'NetworkManager*.rpm'
+ CHECK_RESULT 1 0 0 'Failed option: -c'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: -c'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: -c'
+ LOG_ERROR 'Failed option: -c'
+ message='Failed option: -c'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Failed option: -c'
Sun Dec  4 08:57:54 2022 - ERROR - Failed option: -c
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_rpmdevtools_rpmargs.sh line 62'
+ message='oe_test_rpmdevtools_rpmargs.sh line 62'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_rpmdevtools_rpmargs.sh line 62'
Sun Dec  4 08:57:55 2022 - ERROR - oe_test_rpmdevtools_rpmargs.sh line 62
+ return 0
+ rpmdev-diff -l 'NetworkManager*.rpm' 'NetworkManager-libnm*.rpm'
Error: file does not exist: 'NetworkManager*.rpm'
+ CHECK_RESULT 1 0 0 'Failed option: -l'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: -l'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: -l'
+ LOG_ERROR 'Failed option: -l'
+ message='Failed option: -l'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Failed option: -l'
Sun Dec  4 08:57:56 2022 - ERROR - Failed option: -l
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_rpmdevtools_rpmargs.sh line 64'
+ message='oe_test_rpmdevtools_rpmargs.sh line 64'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_rpmdevtools_rpmargs.sh line 64'
Sun Dec  4 08:57:57 2022 - ERROR - oe_test_rpmdevtools_rpmargs.sh line 64
+ return 0
+ rpmdev-diff -L 'NetworkManager*.rpm' 'NetworkManager-libnm*.rpm'
Error: file does not exist: 'NetworkManager*.rpm'
+ CHECK_RESULT 1 0 0 'Failed option: -L'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: -L'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: -L'
+ LOG_ERROR 'Failed option: -L'
+ message='Failed option: -L'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Failed option: -L'
Sun Dec  4 08:57:57 2022 - ERROR - Failed option: -L
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_rpmdevtools_rpmargs.sh line 66'
+ message='oe_test_rpmdevtools_rpmargs.sh line 66'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_rpmdevtools_rpmargs.sh line 66'
Sun Dec  4 08:57:58 2022 - ERROR - oe_test_rpmdevtools_rpmargs.sh line 66
+ return 0
+ rpmdev-diff -m 'NetworkManager*.rpm' 'NetworkManager-libnm*.rpm'
Error: file does not exist: 'NetworkManager*.rpm'
+ CHECK_RESULT 1 0 0 'Failed option: -m'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: -m'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: -m'
+ LOG_ERROR 'Failed option: -m'
+ message='Failed option: -m'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Failed option: -m'
Sun Dec  4 08:57:59 2022 - ERROR - Failed option: -m
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_rpmdevtools_rpmargs.sh line 68'
+ message='oe_test_rpmdevtools_rpmargs.sh line 68'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_rpmdevtools_rpmargs.sh line 68'
Sun Dec  4 08:58:00 2022 - ERROR - oe_test_rpmdevtools_rpmargs.sh line 68
+ return 0
+ rpmdev-diff -c -y 'NetworkManager*.rpm' 'NetworkManager-libnm*.rpm'
Error: file does not exist: 'NetworkManager*.rpm'
+ CHECK_RESULT 1 0 0 'Failed option: -y'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: -y'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: -y'
+ LOG_ERROR 'Failed option: -y'
+ message='Failed option: -y'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Failed option: -y'
Sun Dec  4 08:58:01 2022 - ERROR - Failed option: -y
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_rpmdevtools_rpmargs.sh line 70'
+ message='oe_test_rpmdevtools_rpmargs.sh line 70'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_rpmdevtools_rpmargs.sh line 70'
Sun Dec  4 08:58:03 2022 - ERROR - oe_test_rpmdevtools_rpmargs.sh line 70
+ return 0
+ rpmdev-extract -q 'NetworkManager*.rpm'
Error: file does not exist: 'NetworkManager*.rpm'
+ test -d 'NetworkManager*riscv64'
+ CHECK_RESULT 1 0 0 'Failed option: -q'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: -q'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: -q'
+ LOG_ERROR 'Failed option: -q'
+ message='Failed option: -q'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Failed option: -q'
Sun Dec  4 08:58:04 2022 - ERROR - Failed option: -q
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_rpmdevtools_rpmargs.sh line 74'
+ message='oe_test_rpmdevtools_rpmargs.sh line 74'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_rpmdevtools_rpmargs.sh line 74'
Sun Dec  4 08:58:05 2022 - ERROR - oe_test_rpmdevtools_rpmargs.sh line 74
+ return 0
+ rpmdev-extract -f 'NetworkManager*.rpm'
+ grep NetworkManager
Error: file does not exist: 'NetworkManager*.rpm'
+ CHECK_RESULT 1 0 0 'Failed option: -f'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: -f'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: -f'
+ LOG_ERROR 'Failed option: -f'
+ message='Failed option: -f'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Failed option: -f'
Sun Dec  4 08:58:06 2022 - ERROR - Failed option: -f
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_rpmdevtools_rpmargs.sh line 76'
+ message='oe_test_rpmdevtools_rpmargs.sh line 76'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_rpmdevtools_rpmargs.sh line 76'
Sun Dec  4 08:58:06 2022 - ERROR - oe_test_rpmdevtools_rpmargs.sh line 76
+ return 0
+ rpmdev-extract -C ./tmp_dir 'NetworkManager*.rpm'
Error: file does not exist: 'NetworkManager*.rpm'
+ test -d './tmp_dir/NetworkManager*riscv64'
+ CHECK_RESULT 1 0 0 'Failed option: -C'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: -C'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: -C'
+ LOG_ERROR 'Failed option: -C'
+ message='Failed option: -C'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Failed option: -C'
Sun Dec  4 08:58:08 2022 - ERROR - Failed option: -C
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_rpmdevtools_rpmargs.sh line 79'
+ message='oe_test_rpmdevtools_rpmargs.sh line 79'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_rpmdevtools_rpmargs.sh line 79'
Sun Dec  4 08:58:09 2022 - ERROR - oe_test_rpmdevtools_rpmargs.sh line 79
+ return 0
+ rpmdev-extract -h
+ grep -A 10 rpmdev-extract
+ grep Options:
Options:
+ CHECK_RESULT 0 0 0 'Failed option: -h'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Failed option: -h'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ rpmdev-extract -v
+ grep 'rpmdev-extract version'
rpmdev-extract version 1.13
+ CHECK_RESULT 0 0 0 'Failed option: -v'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Failed option: -v'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ head -n 1
+ awk '{print $2}'
+ rpmdev-md5 'NetworkManager*rpm'
+ grep 'NetworkManager.*rpm'
md5sum: 'NetworkManager*rpm': No such file or directory
Error: file does not exist: '/root/mugen-riscv/testcases/cli-test/rpmdevtools/oe_test_rpmdevtools_rpmargs/NetworkManager*rpm'
+ CHECK_RESULT 1 0 0 'Failed command: rpmdev-md5'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed command: rpmdev-md5'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed command: rpmdev-md5'
+ LOG_ERROR 'Failed command: rpmdev-md5'
+ message='Failed command: rpmdev-md5'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Failed command: rpmdev-md5'
Sun Dec  4 08:58:11 2022 - ERROR - Failed command: rpmdev-md5
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_rpmdevtools_rpmargs.sh line 86'
+ message='oe_test_rpmdevtools_rpmargs.sh line 86'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_rpmdevtools_rpmargs.sh line 86'
Sun Dec  4 08:58:12 2022 - ERROR - oe_test_rpmdevtools_rpmargs.sh line 86
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Sun Dec  4 08:58:13 2022 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 13
+ test 13 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Sun Dec  4 08:58:14 2022 - ERROR - failed to execute the case.
+ exit 13
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Sun Dec  4 08:58:15 2022 - INFO  - Start to restore the test environment.
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z /tmp/tmp7i32eofi ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile /tmp/tmp7i32eofi
+ '[' 0 -ne 0 ']'
+ rm -rf /ALT ./tmp_dir 'NetworkManager*' 'NetworkManager-libnm*'
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Sun Dec  4 08:58:38 2022 - INFO  - End to restore the test environment.
